examples/mediapipe/BackgroundEnchantments/README.md

```diff
@@ -0,0 +1,39 @@
+# Background Enhancement Sample Application
+This sample application shows how to create background enhancement using [@vonage/ml-transformers](https://www.npmjs.com/package/@vonage/ml-transformers) package.
+# Demo
+You can see a demo of this sample running [here](https://vonage-background-enchantments-sample.s3.amazonaws.com/index.html)
+# Building the application
+**This is very simple**
+## Configure the environment
+    nvm install 16.13
+    nvm use 16.13
+    npm i
+## Run dev version
+
+    npm run dev
+
+## Run production version
+
+    npm run build
+    npm run preview
+
+# Using it in the "real world"
+In our example the result is shown on side canvas.
+However, in the real world you want to use it with [OT.Publisher](https://tokbox.com/developer/guides/vonage-media-processor/js/#publisher-setvideomediaprocessorconnector-method)
+**That is simple too**
```

Comment:
These kind of sentences do not add any value to how the integration is or the library is used in a real use case. I would remove it.